src/components/Pill/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const getPills = (statuses) => {
+  const pills = [];
+  statuses.forEach((status) => {
+    switch (status) {
+      case 'unpublished':
+        pills.push({
+          text: 'Incomplete',
+          className: 'badge badge-danger',
+        });
+        break;
+      case 'reviewed':
+        pills.push({
+          text: 'Scheduled',
+          className: 'badge badge-primary',
+        });
+        break;
+      case 'review_by_legal':
+      case 'review_by_internal':
+        pills.push({
+          text: 'In review',
+          className: 'badge badge-warning',
+        });
+        pills.push({
+          text: <i className="fa fa-lock" />,
+        });
+        break;
+      case 'published':
+        pills.push({
+          text: 'Live',
```

Comment:
Huh, I looked that up and that _is_ Carol's wording. I'm a little surprised that we'd use that for partners. Feels techy to me.